### PR TITLE
Update requirements.md

### PR DESCRIPTION
**Pull request by downTheFallLine**
a typo or two fixed, and some questions?

### CODE REVIEW

File: requirements.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
 * **transport independent** (HTTPS, bluetooth, AMQP, SMTP, NFC, and so forth)
 
 * **routable** (like email, A can talk to B without a direct connection to B, by requesting that a message be forwarded--needed to mix and match transports; also, could pass through a mix network or other infrastructure that isn’t aware of it as anything other than blobs to be moved)
 
 * **secure** (tamper-proof, MITM impossible, composed of primitives widely considered best practice, vetted by experts, has formal security proofs, security guarantees of a given message are known)
 
-* **private** (third parties can’t learn who’s communicating about what, when--broader than “confidentiality” in CIA because metadata like timing and identities of parties is also protected; also, possible for a sender to be anonymous to recipient) 
+* **private** (third parties can’t learn who’s communicating about what, when--broader than “confidentiality” in CIA {huh what does this mean?} because metadata like timing and identities of parties is also protected; also, possible for a sender to be anonymous to recipient)
```
Comment (kdenhartog) [inline requirements.md:11]: CIA is a common term used in security to indicate "Confidentiality, Integrity, Authenticity" the "A" can sometimes mean "Availability" as well.

Here's some good references: https://developer.mozilla.org/en-US/docs/Archive/Security/Confidentiality,_Integrity,_and_Availability

https://security.stackexchange.com/questions/148173/authenticity-confidentiality-integrity-general-questions

File: requirements.md [hunk 2]
```diff
@@ -18,8 +18,8 @@
 
 * **platform-independent** (usable on embedded, not just servers+desktops+mobile)
 
-* **composable** (pick the crypto suite and features you need, without incurring overwhelming dependencies for stuff you don’t)
+* **composable** (pick the crypto suite and features you need, without incurring overwhelming dependencies for stuff you don’t) {this would seemingly require some form of negotiation, similar to what happens in TLS.   What if the sender decided to cipher the message with SecretSquirrelLevel10 algo and the recipient cant handle that algo for any number of reasons (export controls, etc)?}
```
Comment (kdenhartog) [inline requirements.md:21]: This would be a good comment to raise in the issue, but I don't think we need to add it to the spec docs.

**Review by kdenhartog — CHANGES_REQUESTED** [review]
Let's remove the {} comments and move those discussions to issues, but happy to get the typos fixes in.

File: requirements.md (hunk 1 above)
Comment (downTheFallLine) [reply requirements.md:11]: Yes, I understand.   CIA, and many other TLA's have multiple or common meanings.  When using an acronym in a technical document, its best practice to define it upon first use.   I'll make that edit.

Comment (kdenhartog) [reply requirements.md:11]: Make sense, I saw your PR that added it and approved. Good work!

**Comment by TelegramSam** [comment]
moved to issues

**Pull request closed without merging**